docs/fastdds/dds_layer/core/policy/policy.rst

```diff
@@ -1,5 +1,2259 @@
+^^^^^^^^^^^^^^^^^^^^
+
+This QoS Policy indicates the level of reliability offered and requested by the service.
+
+List of QoS Policy data members:
+
++-----------------------------------+--------------------------------+----------------------------------+
+| Data Member Name                  | Type                           | Default Value                    |
++===================================+================================+==================================+
+|                                   |                                | BEST_EFFORT for DataReaders      |
+| kind                              | :ref:`reliabilityqospolicykind`| :raw-html:`<br />`               |
+|                                   |                                | RELIABLE for DataWriters         |
++-----------------------------------+--------------------------------+----------------------------------+
+| max_blocking_time                 | fastrtps::Duration_t           | 100 ms                           |
++-----------------------------------+--------------------------------+----------------------------------+
+
+* **Kind**: Specifies the behavior of the service regarding delivery of the samples.
+  See :ref:`reliabilityqospolicykind` for further details.
+* **Max blocking time**: Configures the maximum duration that the write operation can be blocked.
+
+.. note::
+     This QoS Policy concerns to Topic, DataWriter and DataReader entities.
+     :raw-html:`<br />`
+     It cannot be changed after the creation of the entity.
+
+.. warning::
+    For DataWriters and DataReaders to match, they must follow the compatibility rule. See :ref:`reliability_compatibilityrule`
+    for further details.
+
+.. _reliabilityqospolicykind:
+
+ReliabilityQosPolicyKind
+""""""""""""""""""""""""
+
+There are two possible values:
+
+* **Best Effort**: It indicates that it is acceptable not to retransmit the missing samples, so the messages are sent
+  without waiting for an arrival confirmation.
+  Presumably new values for the samples are generated often enough that it is not necessary to re-send any sample.
+  However, the data samples sent by the same DataWriter will be stored in the DataReader history in the same order they
+  occur.
+  In other words, even if the DataReader miss some data samples, a newer value will never be change to an older value.
+* **Reliable**: It indicates that the service will attempt to deliver all samples of the DataWriter's history expecting
+  an arrival confirmation from the DataReader.
+  The data samples sent by the same DataWriter cannot be made available to the DataReader if there are previous samples
+  that have not been received yet.
+  The service will retransmit the lost data samples in order to reconstruct a correct snapshot of the DataWriter
+  history before it is accessible by the DataReader.
+
+  This option may block the write operation, hence the `max_blocking_time` is set that will unblock it once the time
+  expires.
```

Comment:
Explain that the write operation fails if the max_blocking_time expires before actually sending the data